npm/angular/src/mount.ts

```diff
@@ -260,6 +262,11 @@ function setupFixture<T> (
 
   fixture.whenStable().then(() => {
     fixture.autoDetectChanges(config.autoDetectChanges ?? true)
+  }).catch((e) => {
+    // If this promise does not settle in Angular 19 it is rejected
+    // https://github.com/angular/angular/blob/main/CHANGELOG.md#1900-2024-11-19
+    // eslint-disable-next-line no-console
```

Comment:
we likely don't want to propagate this failure up and fail, just log it to the console. some users trigger stability manually for some difference use cases (see https://github.com/cypress-io/cypress/issues/30327)